tests/transforms/test_batch_inputs.py

```diff
@@ -1,129 +0,0 @@
-# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
```

Comment:
I agree that this file should be deleted. Just to make sure, @josh146 is this fine? :)

Reply:
@josh146 could you quickly give your thoughts on deleting this duplicated test file before merging this PR?